recipe/meta.yaml

```diff
@@ -28,6 +34,7 @@ requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
+    - cuda-version {{ cuda_compiler_version }}  # [cuda_major >= 12.0] 
```

Comment:
As this already gets [pulled in by the complier]( https://github.com/conda-forge/cuda-nvcc-impl-feedstock/blob/4199d2047d7ada2c7c2c1fdb84f0fe57f860cd34/recipe/meta.yaml#L94 ), think we can drop this

```suggestion
```